src/renderer/base/RenderEngineBase.cpp

```diff
@@ -7,12 +7,6 @@
 using namespace Microsoft::Console;
 using namespace Microsoft::Console::Render;
 
-RenderEngineBase::RenderEngineBase() :
-    _titleChanged(false),
-    _lastFrameTitle(L"")

```

Comment:
(double checking) in the .hpp, `_lastFrameTitle` isn't set to anything. But STL types like this just automatically instantiate it anyways right? (unlike WinRT objects?)

Reply:
Yeah, practically all STL objects instantiate to "empty" by default.

I mean, WinRT sometimes does the same (`winrt::com_ptr`, `winrt::hstring`, really anything under `winrt` directly), but "oh oops" except for all those projected types, because consistency. 😄